src/services/users.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const { v4: uuidv4 } = require('uuid');
+
+let users = [];
+
+function getAllUsers() {
+  return users;
+}
+
+function getUserById(userId) {
+  const foundUser = users.find(user => user.id === +userId);
+
+  return foundUser || null;
+}
+
+function createUser(name) {
+  const newUser = {
+    id: parseInt(uuidv4(), 16),
```

Comment:
Change 'parseInt(uuidv4(), 16)' to 'uuidv4()' as UUIDs should be kept as strings and not be converted to numbers.